internal/cli/capact/components.go

```diff
@@ -0,0 +1,496 @@
+	return errors.Wrap(err, "while annotating secret")
+}
+
+func (c *Capact) InstallUpgrade(version string) error {
+	upgradeCli := c.upgradeAction(version)
+
+	capactValues := c.opts.Parameters.Override.CapactValues
+	if version == LocalVersionTag {
+		capactValues.Global.ContainerRegistry.Path = LocalDockerPath
+		capactValues.Global.ContainerRegistry.Tag = LocalDockerTag
+	}
+	s := structs.New(capactValues)
+	s.TagName = "json"
+	mappedValues := s.Map()
+
+	if c.opts.Environment == KindEnv {
+		values, err := ValuesFromString(capactKindOverridesYaml)
+		if err != nil {
+			return errors.Wrap(err, "while converting override values")
+		}
+		mappedValues = tools.MergeMaps(values, mappedValues)
+	}
+	return c.runUpgrade(upgradeCli, mappedValues)
+}
+
+func (m *Monitoring) InstallUpgrade(version string) error {
+	upgradeAction := m.upgradeAction(version)
+
+	values := map[string]interface{}{}
+	return m.runUpgrade(upgradeAction, values)
+}
+
+func GetActionConfiguration(k8sCfg *rest.Config, forNamespace string) (*action.Configuration, error) {
+	actionConfig := new(action.Configuration)
+	helmCfg := &genericclioptions.ConfigFlags{
+		APIServer:   &k8sCfg.Host,
+		Insecure:    &k8sCfg.Insecure,
+		CAFile:      &k8sCfg.CAFile,
+		BearerToken: &k8sCfg.BearerToken,
+		Namespace:   ptr.String(forNamespace),
+	}
+
+	debugLog := func(format string, v ...interface{}) {
+		// noop
+	}
+
+	err := actionConfig.Init(helmCfg, forNamespace, "secrets", debugLog)
+
+	if err != nil {
+		return nil, errors.Wrap(err, "while initializing Helm configuration")
+	}
+
+	return actionConfig, nil
+}
+
+type Helm struct {
+	configuration *action.Configuration
+	opts          Options
+}
+
+func NewHelm(configuration *action.Configuration, opts Options) *Helm {
+	if opts.Parameters.IncreaseResourceLimits {
+		opts.Parameters.Override.CapactValues.Gateway.Resources = IncreasedGatewayResources()
+		opts.Parameters.Override.CapactValues.HubPublic.Resources = IncreasedHubPublicResources()
+		opts.Parameters.Override.Neo4jValues.Neo4j.Core.Resources = IncreasedNeo4jResources()
+	}
+	if opts.Parameters.Override.HelmRepoURL == LatestVersionTag {
+		opts.Parameters.Override.HelmRepoURL = HelmRepoLatest
+	}
+
+	return &Helm{configuration: configuration, opts: opts}
+}
+
+func (h *Helm) InstallComponnents(w io.Writer, status *printer.Status) error {
+	var err error
+	helper := NewHelmHelper()
+
+	for _, component := range components {
+		if shouldSkipTheComponent(component.Name(), h.opts.SkipComponents) {
+			continue
+		}
+
+		component.withOptions(&h.opts)
+		component.withConfiguration(h.configuration)
+		component.withWriter(w)
+
+		version := h.opts.Parameters.Version
+		if version == LatestVersionTag {
+			version, err = helper.GetLatestVersion(h.opts.Parameters.Override.HelmRepoURL, component.Name())
+			if err != nil {
+				return errors.Wrapf(err, "while getting latest version for %s", component.Name())
+			}
+		}
+
+		status.Step("Installing %s Helm Chart", component.Name())
```

Comment:
```suggestion
		status.Step("Installing %s Helm chart", component.Name())
```
AFIK only the `Helm` name should be capitalized